pkg/jobrunaggregator/jobrunaggregatorlib/spyglass_summary.go

```diff
@@ -73,7 +73,14 @@ body {
 				if prowJob.Status.CompletionTime != nil {
 					duration = prowJob.Status.CompletionTime.Sub(prowJob.Status.StartTime.Time)
 				}
-				html += fmt.Sprintf(" %v after %v\n", prowJob.Status.State, duration)
+				// Create a string of asterisks proportional to the number of seconds (scaled down by 1000)
+				// so we can visually ascertain relative job time.
+				scaleSeconds := int(duration.Seconds() / 1000.0)
+				var stars string
+				for j := 0; j < scaleSeconds; j++ {
```

Comment:
I think strings.Repeat() does this for you.

Reply:
Nice and thanks!  I pushed the change.